documentation/source/newsfragments/2041.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix usage of :func:`~cocotb.util.hexdump` and :func:`~cocotb.util.hexdiff` in the :class:`~cocotb.scoreboard.Scoreboard`.
```

Comment:
Should we add a [``.. deprecated::``](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated) here already? 
Some short discussion about it in #1642.

Reply:
No, that should probably go in the `Scoreboard` class itself.

Reply:
I added it to the scoreboard and everything in cocotb.generators, but only the scoreboard is in the documentation.

Reply:
#2042 adds generators to the docs so that they can be visibly deprecated.